docs/contributing.md

```diff
@@ -114,6 +114,7 @@ The suffixes are:
 `release-static` is symlinked to `default`, so that commands like `molecule test` will use it.
 
 Intentional other differences between tests:
```

Comment:
Please document it under here also.

Even though it is not correlated with the names of the tests, we should still list that some have Apache and others havings nginx.